boards/arm/stm32/nucleo-f103rb/src/stm32_appinit.c

```diff
@@ -24,27 +24,16 @@
 
 #include <nuttx/config.h>
 
-#include <sys/types.h>
-#include <syslog.h>
-
 #include <nuttx/board.h>
-#include <nuttx/leds/userled.h>
 
 #include "nucleo-f103rb.h"
 
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
 
-#undef HAVE_LEDS
-#undef HAVE_DAC
-
-#if !defined(CONFIG_ARCH_LEDS) && defined(CONFIG_USERLED_LOWER)
-#  define HAVE_LEDS 1
-#endif
-
-#if defined(CONFIG_DAC)
-#  define HAVE_DAC 1
+#ifndef OK
```

Comment:
why need this?

Reply:
It is a copy-paste from nucleo-f302r8. I'm not sure what story is behind it, but this is present in many boards. Probably some configuration options re-define OK for xxx_appinit.c